src/Licenses/LicensesClient.php

```diff
@@ -128,4 +135,81 @@ public function find(array $postData, int $perPage = 100, int $page = 1, array $
 
         return new FindResult($response, $this, $postData, $parameters);
     }
+
+    /**
+     * @param string $reference
+     *
+     * @return string
+     *
+     * @throws GuzzleException
+     * @throws NotFoundException
+     * @throws PublicApiClientException
+     */
+    public function getConfigsRaw(string $reference): string
+    {
+        $this->path = '/' . $reference . self::CONFIGS_PATH;
+
+        return $this->get();
+    }
+
+    /**
+     * @param string $reference
+     *
+     * @return Generator
```

Comment:
```@return Generator|Config[]```

Reply:
Psalm will throws an error if we do this because it mismatch the declared function return type